homeassistant/components/wevolor/config_flow.py

```diff
@@ -0,0 +1,72 @@
+"""Config flow for Wevolor Control for Levolor Motorized Blinds integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pywevolor import Wevolor
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import CONFIG_CHANNELS, CONFIG_HOST, CONFIG_TILT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONFIG_HOST): str,
+        vol.Required(CONFIG_TILT, default=False): bool,
+        vol.Required(CONFIG_CHANNELS, default=6): vol.All(int, vol.Range(min=1, max=6)),
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str, Any]:
+    """Validate the user input allows us to connect."""
+
+    wevolor = Wevolor(data[CONFIG_HOST])
+    status = await hass.async_add_executor_job(wevolor.get_status)
```

Comment:
`Wevolor.get_status()` is async so you shouldn't use `async_add_executor_job`.

```suggestion
    status = await wevolor.get_status()
```